libraries/Microsoft.Bot.Builder/Skills/SkillHandler.cs

```diff
@@ -19,6 +19,8 @@ namespace Microsoft.Bot.Builder.Skills
     /// </summary>
     public class SkillHandler : ChannelServiceHandler
     {
+        public const string SkillConversationRefenenceKey = "SkillConversationReference";
```

Comment:
Should we namespace this one to avoid collisions? Something like 

```
public static readonly string SkillConversationRefenenceKey = $"{typeof(SkillHandler).Namespace}.SkillConversationReference";
```